### PR TITLE
fix: species component no key

### PR DESCRIPTION
**Pull request by RubenSmn**
# Description
Fixes warning of component having no key

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/TreeSpeciesCard.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function TreeSpeciesCard(props) {
       sx={{
         borderRadius: 4,
         overflow: 'hidden',
-        mb: [2, 4],
+        my: [2, 4],
```
Comment (dadiorchen) [inline src/components/TreeSpeciesCard.js:24]: @RubenSmn let's remove these margin from the component and let the client to deal with the spacing, in this way, we make the component more re-useable.

Comment (dadiorchen) [reply src/components/TreeSpeciesCard.js:24]: So, remove margin from compnent and put it into the invoker of this component

**Comment by RubenSmn** [comment]
@dadiorchen took another approach and fixed it!

File: src/pages/organizations/[organizationid].js [hunk 2]
```diff
@@ -367,14 +367,14 @@ export default function Organization(props) {
                 ))}
                 {/* Placeholder, remove after API fixed */}
                 {organization?.species?.species?.map((s) => (
-                  <>
+                  <React.Fragment key={s.name}>
```
Comment (dadiorchen) [inline src/pages/organizations/[organizationid].js:370]: I guess this is the same as: `<></>` ? right?

Comment (RubenSmn) [reply src/pages/organizations/[organizationid].js:370]: Yes but this way you can pass the key

**Pull request merged**